src/atopile/kicad_plugin/reloadgroup.py

```
@@ -74,8 +74,13 @@ def Run(self):
             # Add all items to the group
             # Start with the footprints
             for fp_addr in expected_footprints - footprints_in_group:
-                g.AddItem(footprints[fp_addr])
-                log.debug(f"Added footprint {fp_addr}")
+                if fp_addr in footprints:
```

While the error was shit, this is a silent failure. I don't think this is an upgrade